docs/snakefiles/reporting.rst

```diff
@@ -246,36 +254,77 @@ Further, using ``url_args()`` you can add URL arguments and using ``anchor()`` y
 .. code-block:: bash
 
-    snakemake --report report.html
+    snakemake --report
 
-after your workflow has finished.
-All other information contained in the report (e.g. runtime statistics) is automatically collected during creation.
-These statistics are obtained from the metadata that is stored in the ``.snakemake`` directory inside your working directory.
+after your workflow has finished successfully.
+This will implicitly assume a report file name of ``report.html`` and will create this self-contained HTML report.
+If you want to give your report a custom name, you can do so by specifying this on the command-line:
+
+.. code-block:: bash
+
+    snakemake --report my_report.html
+
+As specifying an HTML file for the report will always embed any user-defined report outputs into the same HTML file, this report output type is only suitable for smaller reports.
 
+.. _snakefiles-self_contained_zip_archive:
 
-You can define an institute specific stylesheet with:
+Self-contained ZIP archive
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For anything more complex, it is recommended to generate a ``ZIP`` archive report.
+For example, if you have lots of samples you report on, or if you specified lots of different workflow outputs for report inclusion.
+To get such a self-contained ``ZIP`` archive, simply specify a ``.zip`` file name instead:
 
 .. code-block:: bash
 
-    snakemake --report report.html --report-stylesheet custom-stylesheet.css
+    snakemake --report some_report.zip
 
-In particular, this allows you to e.g. set a logo at the top (by using CSS to inject a background for the placeholder ``<div id="brand">``, or overwrite colors.
-For an example custom stylesheet defining the logo, see :download:`here <../../tests/test_report/custom-stylesheet.css>`.
-The report for above example can be found :download:`here <../../tests/test_report/expected-results/report.html>` (with a custom branding for the University of Duisburg-Essen).
-The full example source code can be found `here <https://github.com/snakemake/snakemake/tree/main/tests/test_report/>`__.
+You can move this file wherever you need it and then unpack it.
+The main entry point is always the ``report.html`` file in the main folder that this creates (for the above example, this folder will be named ``some_report``).
+Just open that file in a web browser and explore your results.
 
-Note that the report can be restricted to particular jobs and results by specifying targets at the command line, analog to normal Snakemake execution.
+Partial reports
+^^^^^^^^^^^^^^^
+
+The report can be restricted to particular jobs and results by specifying targets at the command line, analog to normal Snakemake execution.
 For example, with
 
 .. code-block:: bash
 
     snakemake fig1.svg --report report-short.html
 
 the report contains only ``fig1.svg``.
+This can for example be useful when a larger workflow has not yet run to completion, but you already want to explore some intermediate outputs in the report.
```

Comment:
Erase the qualifier "for example" - no need.

Reply:
```suggestion
This can be useful when a larger workflow has not yet run to completion, but you already want to explore some intermediate outputs in the report.
```